tests/queries/0_stateless/02151_http_s_structure_set_eof.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+tmp_file=$(mktemp "$CURDIR/clickhouse.XXXXXX.csv")
+trap 'rm $tmp_file' EXIT
+
+# NOTE: this file should be huge enough, so that it is impossible to upload it
+# in 0.15s, see timeout command below, this will ensure, that EOF will be
+# received during creating a set from externally uploaded table.
+#
+# Previously code there wasn't ready for EOF, and you will get one of the
+# following assertions:
+#
+#     - ./src/IO/ReadBuffer.h:58: bool DB::ReadBuffer::next(): Assertion `!hasPendingData()' failed.
+#     - ./src/Server/HTTP/HTMLForm.cpp:245: bool DB::HTMLForm::MultipartReadBuffer::skipToNextBoundary(): Assertion `boundary_hit' failed.
+#     - ./src/IO/LimitReadBuffer.cpp:17: virtual bool DB::LimitReadBuffer::nextImpl(): Assertion `position() >= in->position()' failed.
```

Comment:
Probably this notice about assertions won't be useful in the future in our code base, so maybe it's better to remove it.

Reply:
I would prefer to keep it, at least it does not interfere anything (and this will help in investigating similar issues)